modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -126,7 +128,25 @@ instead.
 `CountBadge` now uses `createComponent`, Canvas Tokens, and our new styling utilities. The component
 API has not been changed, and it should behave identically as it did in previous versions.
 
-### FormField Preview
+### Text
+
+`Text` has been refactored to use our new styling utilities and utilize new tokens and no longer
+extends `Box` component. The component API has not been changed, and it should behave identically as
+it did in previous versions and support Emotion `styled` and style props. Type level components,
+`Title`, `Heading`, `BodyText` and `Subtext`, have not been changed, but because they extend `Text`
+components, all changes made for `Text` will be applied for them as well.
+
+### Label Text
+
+`LabelText` has been refactored to use our new styling utilities and utilize new tokens and no
+longer extends `Text` component. The component API is similar to previous one and it still supports
+`variant`, `disabled` and `Box` styles props. `typeLevel` prop has been removed from `LabelText` and
+we suggest to use type level components, `Title`, `Heading`, `BodyText` and `Subtext`, instead.
+
+🤖 The codemod will handle a `typeLevel` removal and replacement by `Text` component for you
+automatically.
+
```

Comment:
So they should only use `LabelText` when they don't want to change the size? otherwise, just use `text` if they need more flexibility?